src/components/TopBar/TopBarMenu.vue

```diff
@@ -377,7 +382,40 @@ export default {
 
 		showCallLayoutSwitch() {
 			return !this.$store.getters.isEmptyCallView
-		}
+		},
+
+	},
+
+	watch: {
+
+		'model.attributes.speaking'(speaking) {
+			// user stops speaking in lowerHandTimeout
+			if (this.lowerHandTimeout !== null && !speaking) {
+				this.lowerHandDelay = AUTO_LOWER_HAND_THRESHOLD - (Date.now() - this.speakingTimestamp)
```

Comment:
I'm afraid of getting a negative value here. Although it may be treated as 0 by `setTimeout` function, I'd prefer to wrap it with `Math.max`. Also, the value is not accumulating properly (which was intended, as I get from discussion):

```suggestion
				this.lowerHandDelay = Math.max(0, this.lowerHandDelay - (Date.now() - this.speakingTimestamp))
```

Before | After
-- | --
![Screenshot from 2024-06-18 11-02-09](https://github.com/nextcloud/spreed/assets/93392545/aca0e232-fb9a-4925-89fb-f7f1b425046a) |  ![image](https://github.com/nextcloud/spreed/assets/93392545/ca2e03c2-868d-46c3-8e65-810b1247d66d)